charts/geth/templates/_cmd.tpl

```diff
@@ -9,7 +9,7 @@
   . /env/init-nodeport;
 {{- end }}
   exec geth
-  --datadir=/data
+  --datadir=/data/geth/
```

Comment:
This will be a breaking change for existing installs - is there a reason why its needed across all the client charts? This sort of concern is usually handled at the PVC layer :)

Reply:
Fixed.